src/sentry/static/sentry/app/views/stream/index.jsx

```diff
@@ -123,9 +127,6 @@ const Stream = createReactClass({
         },
         this.fetchData
       );
-    } else {
-      // something else has changed, just refetch
```

Comment:
what's an example of "something else"  that was causing unnecessary fetches?

Reply:
When search id or url is changed this would be called (but it's already handled on the lines this below this)